ethpm_cli/parser.py

```diff
@@ -117,13 +120,32 @@ def add_alias_arg_to_parser(parser: argparse.ArgumentParser) -> None:
 #
 
 
-# todo: extend to pin & release a local manifest
 def release_cmd(args: argparse.Namespace) -> None:
     config = Config(args)
-    release_package(args.package_name, args.package_version, args.manifest_uri, config)
+
+    manifest_uri = args.manifest_uri
+
+    package_name = args.package_name
+    package_version = args.package_version
```

Comment:
If we're releasing local manifest, then I think we should make the args (`manifest_uri`, `package_name`, `package_version`) and (`manifest_path`) mutually exclusive groups (like we do with `ethpm get` on line 589). We just want to avoid any confusion since the package info found in the manifest & the newly generated hash will override whatever the user passes in.